_users/metrics-in-1.0.0-m1.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Metrics in 1.0.0-m1"
+lang: en
+ref: metrics
+permalink: /users/metrics-in-1.0.0-m1/
+excerpt: "Metrics in 1.0.0-m1"
+last_modified_at: 2017-12-30T10:01:43-04:00
+redirect_from:
+  - /theme-setup/
+---
+
+{% include toc %}
+Metrics had supported from Java chassis version 0.5.0,in version 1.0.0-m1,we had reconstruction it and add some more features,please subscribe ServiceComb mail-list(dev-subscribe@servicecomb.incubator.apache.org) in order to get latest news.
```

Comment:
Please checkout the user guide and release note (link) for more information.  (We don't suggest user to find how to use the software by going through the email)

Reply:
Done